### PR TITLE
:sparkles: make mqtt ca config optional

### PR DESCRIPTION
**Pull request by skeeey**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/cloudevents/generic/options/mqtt/options.go [hunk 1]
```diff
@@ -345,3 +334,26 @@ func getAgentPubTopic(ctx context.Context) (*PubTopic, error) {
 
 	return nil, fmt.Errorf("invalid agent pub topic")
 }
+
+func rootCAs(caFile string) (*x509.CertPool, error) {
```
Comment (qiujian16) [inline pkg/cloudevents/generic/options/mqtt/options.go:338]: maybe add a comment on this function?

Comment (skeeey) [reply pkg/cloudevents/generic/options/mqtt/options.go:341]: added

**Pull request merged**